cms/taxonomy/tests/test_management_commands.py

```diff
@@ -0,0 +1,521 @@
+# pylint: disable=too-many-public-methods
+from collections.abc import Iterable
+from typing import Any
+from unittest.mock import Mock, patch
+
+import requests
+from django.test import TestCase
+from requests import HTTPError
+
+from cms.taxonomy.management.commands import sync_topics
+from cms.taxonomy.models import Topic
+
+
+class SyncTopicsTests(TestCase):
+    def setUp(self):
+        self.requests_patcher = patch("cms.taxonomy.management.commands.sync_topics.requests")
+        self.mock_requests = self.requests_patcher.start()
+
+    def tearDown(self):
+        self.mock_requests.stop()
+
+    def test_sync_no_topics(self):
+        # Given
+        mock_response = mock_successful_json_response([])
+        self.mock_requests.get.return_value = mock_response
+
+        # When
+        sync_topics.Command().handle()
```

Comment:
Issue: calling management commands should be done using [`call_command`](https://docs.djangoproject.com/en/5.1/ref/django-admin/#django.core.management.call_command)

Reply:
Good spot! Done!